app/src/main/AndroidManifest.xml

```diff
@@ -118,7 +118,8 @@
     </activity>
     <activity
       android:name=".app.story.StoryActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/story_activity_label" />
```

Comment:
It requires reformatting. Also, we are now naming these labels as "title". So you can rename the string to "story_activity_title".

Reply:
+1